lib/util/file.js

```diff
@@ -504,18 +504,10 @@ const File = {
      * @returns {Promise<void>} -
      */
     async saveConfigFile(properties) {
-        const auth = { credentials: {} };
-        const config = properties;
-        for (const cred in config.credentials) {
-            auth.credentials[cred] = {};
-            // copy eid as well to make sure we can test for equality when merging the files during runtime
-            auth.credentials[cred].eid = config.credentials[cred].eid;
-        }
         // we want to save to save the full version here to allow us to upgrade configs properly in the future
-        config.version = packageJson.version;
+        properties.version = packageJson.version;
 
-        await this.writeJSONToFile('', Util.configFileName.split('.json')[0], config);
-        await this.writeJSONToFile('', Util.authFileName.split('.json')[0], auth);
```

Comment:
looks to me like you removed storing credentials in a file completely after all? thought we agreed not to push forward with that in this release?

Reply:
It does still save just via the auth handler, to ensure that this logic is handled discretely within that class (auth is managed in auth class). Happy to discuss moving it out and into this, but it also means continued overloading of the file class.